ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,60 @@
+<!-- Thank you for using Flutter!
+
+     If you are looking for support, please check out our documentation
+     or consider asking a question on Stack Overflow:
+      * https://flutter.io/
+      * https://docs.flutter.io/
+      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill our the template below. Please read
+     our guide to filing a bug first:  https://flutter.io/bug-reports/
+
+-->
+
 ## Steps to Reproduce
 
-Please tell us what you were doing and what went wrong. If you are running flutter tools from the command line, please try adding the `-v` or `--verbose` option to gather more information.
+<!--
+
+     Please tell us exactly how to reproduce the problem you are running into.
+
+     Please attach a small application (ideally just one main.dart file) that
+     reproduces the problem.
 
-If the problem is with your application's rendering, please attach a screenshot and any relevant source code.
-If you are getting an exception in the logs, and your code is implicated in the first few frames, then please include the source code for the functions involved.
+     If the problem is with your application's rendering, then please attach
+     a screenshot and explain what the problem is.
+
+-->
+ 
+1. ...
+2. ...
+3. ...
 
 ## Logs
 
-Run your application with `flutter run` and attach all the log output.
+<!--
+
+      Run your application with `flutter run --verbose` and attach all the
```

Comment:
Many of the issues that are filed are with people unable to get to the point of running.  How about instead:

```
flutter --verbose <command>
```